test/integration/features/basic.feature

```diff
@@ -0,0 +1,20 @@
+# 
+Feature: Bringup
+  In order to use MiniShift
+  As a user
+  I need to be able to bring up a test environment
+
+  Scenario: Docker environment variables
+    Given a "developer"
+     When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+      And should be running
+     Then should have a valid IP address
+     Then executing "minishift ssh echo hello" should contain
+      """
+      hello
+      """
+     Then executing "minishift ssh cat /var/lib/boot2docker/profile" should contain
```

Comment:
I would prefer going for the simpler approach where there is no index for the execution (and only the last command and status is kept). If we really need a scenario that needs to run two commands, we could improve on this...

The reasoning for me is that the Gherkin should be as minimal as possible, unless we need to perform those actions. It is easier to add something to the definition, than removing it later when we do not use it.

Feedback from QA/QE would be appreciated... else, we will have to make the decision.